gcp/src/main/java/org/apache/iceberg/gcp/gcs/GCSInputStream.java

```diff
@@ -117,7 +117,7 @@ public int read() throws IOException {
     readBytes.increment();
     readOperations.increment();
 
-    return singleByteBuffer.array()[0];
+    return singleByteBuffer.array()[0] & 0xFF;
```

Comment:
Should we use `Byte.toUnsignedInt()` here to be more clear?

Reply:
+1